internal/bridge/crunchybridgecluster/postgres_test.go

```diff
@@ -20,16 +20,10 @@ import (
 )
 
 func TestGeneratePostgresRoleSecret(t *testing.T) {
-	tClient := setupKubernetes(t)
-	require.ParallelCapacity(t, 0)
-
-	reconciler := &CrunchyBridgeClusterReconciler{
-		Client: tClient,
-		Owner:  "crunchybridgecluster-controller",
-	}
+	reconciler := &CrunchyBridgeClusterReconciler{}
```

Comment:
Kind of want to rename `generatePostgresRoleSecret` `generatePostgresRoleSecretIntent` or something

Reply:
Yeah, there's a few funcs here that, from the name, I would've said call kubernetes.